recipes-kernel/linux/linux-hikey-aosp_4.4.bb

```diff
@@ -71,4 +71,6 @@ do_configure() {
     bbplain "Saving defconfig to:\n${B}/defconfig"
     oe_runmake -C ${B} savedefconfig
     cp -a ${B}/defconfig ${DEPLOY_DIR_IMAGE}
+    bbplain "Saving .config to:\n${B}/config"
+    cp -a ${B}/.config ${DEPLOY_DIR_IMAGE}/config
```

Comment:
The kernel tree has a script to inspect it off-line: https://github.com/torvalds/linux/blob/master/scripts/extract-ikconfig